### PR TITLE
INGM-389 Safety processes examples

### PR DESCRIPTION
**Pull request by MartinALL91**
### Description

Add STO deactivate example for Safe drives.

Fixes # INGM-389

### Type of change

Please add a description and delete options that are not relevant.

- Add the STO activation and STO deactivation methods.
- Add example script.

### Documentation

Please update the documentation.

- [x] Update docstrings of every function, method or class that change.
- [x] USe [type hints](https://docs.python.org/3/library/typing.html) for every function and argument.
- [x] Add the changes at the `[Unreleased]` section of the [CHANGELOG](CHANGELOG.md).

### Code formatting

- [x] Use black package to format the code: `black -l 100 ingeniamotion tests`. It is recommended to configure the code editor to automatically format the code using black with a max length line of 100.

### Others

- [x] Set fix version field in the Jira issue.

### CODE REVIEW

File: examples/safety_toque_off.py
[the remark this answers is not in the document]
Comment (MartinALL91) [reply examples/safety_toque_off.py:1]: Done

**Review by pzinemanas — APPROVED** [review]

File: ingeniamotion/fsoe.py [hunk 1]
```diff
@@ -69,6 +69,14 @@ def set_reply(self) -> None:
         to the FSoE master handler."""
         self.__master_handler.set_reply(self.safety_slave_pdu_map.get_item_bytes())
 
+    def sto_deactivate(self) -> None:
+        """Set the STO command to deactivate the STO"""
+        self.__master_handler.dictionary.set("STO_COMMAND", True)
```
Comment (GullinBustin) [inline ingeniamotion/fsoe.py:74]: Do not use a literal

File: ingeniamotion/fsoe.py [hunk 2]
```diff
@@ -69,6 +69,14 @@ def set_reply(self) -> None:
...
+
+    def sto_activate(self) -> None:
+        """Set the STO command to activate the STO"""
+        self.__master_handler.dictionary.set("STO_COMMAND", False)
```
Comment (GullinBustin) [inline ingeniamotion/fsoe.py:78]: Do not use a literal

File: ingeniamotion/fsoe.py [hunk 3]
```diff
@@ -52,7 +52,7 @@ def _configure_master(self) -> None:
     def _map_outputs(self) -> None:
         """Configure the FSoE master handler's SafeOutputs."""
         # Phase 1 mapping
-        self.__master_handler.master.dictionary_map.add_by_key("STO_COMMAND", bits=1)
+        self.__master_handler.master.dictionary_map.add_by_key(0x040, bits=1)
```
[the remark this answers is not in the document]
Comment (MartinALL91) [reply ingeniamotion/fsoe.py]: Done

File: examples/safety_torque_off.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    mc.motion.motor_enable()
+    # Wait for the motor to reach a certain velocity (10 rev/s)
+    with contextlib.suppress(IMTimeoutError):
+        mc.motion.wait_for_velocity(velocity=10, timeout=5)
+    # Stop the FSoE master handler
+    mc.fsoe.stop_master(stop_pdos=True)
+    # Disable the motor
+    mc.motion.motor_disable()
+    # Restore the operation mode
+    mc.motion.set_operation_mode(current_operation_mode)
```
Comment (polfeliu) [inline examples/safety_torque_off.py:31]: Is it really important for examples to restore the initial config?

Comment (MartinALL91) [reply examples/safety_torque_off.py:31]: Not really but it doesn't hurt to do it.

File: examples/safety_torque_off.py [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    current_operation_mode = mc.motion.get_operation_mode()
+    # Set the Operation mode to Velocity
+    mc.motion.set_operation_mode(OperationMode.VELOCITY)
+    # Create and start the FSoE master handler
+    mc.fsoe.create_fsoe_master_handler()
+    mc.fsoe.start_master(start_pdos=True)
+    # Deactivate the STO and enable the motor
+    mc.fsoe.sto_deactivate()
+    mc.motion.motor_enable()
+    # Wait for the motor to reach a certain velocity (10 rev/s)
```
[the remark this answers is not in the document]
Comment (MartinALL91) [reply examples/safety_torque_off.py:23]: I tried to make it as simple as possible, just to check that the motor is able to move. There is another more extensive [example ](https://github.com/ingeniamc/ingeniamotion/blob/0.8.0/examples/velocity_torque_ramp.py) of how to move the motor.

[the remark this answers is not in the document]
Comment (MartinALL91) [reply examples/safety_torque_off.py:23]: Done

**Review by GullinBustin — APPROVED** [review]

**Review by polfeliu — APPROVED** [review]

**Pull request merged**